src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,513 @@
+"""Table definitions for EIA AEO resources."""
+
+from typing import Any
+
+# 2024-04-24: workaround so we can *define* a bunch of schemas without
+# documenting them or expecting assets to exist
+_STAGING_RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_eiaaeo__electric_sector_generation_by_technology": {
```

Comment:
@cmgosnell I see you named your ATB tables `core_nrelatb__yearly_projection_*`, should I add `yearly_projection` to my table names too?

Reply:
I did it, it's an important distinction and otherwise we'd rely on people knowing what AEO is.